styleguide/_themes/global/scss/fonts.scss

```diff
@@ -20,7 +21,7 @@ $code: Tahoma, Arial, Geneva, Helvetica;
 }
 
 @mixin copy($color: $gray-darkest) {
-  color: $gray-darkest;
+  color: $color;
```

Comment:
copy should never be any other color. If we want to do this we should define a $brand-copy or $font-base variable.